backtrace-library/src/main/java/backtraceio/library/models/json/BacktraceReport.java

```diff
@@ -185,7 +185,11 @@ public BacktraceReport(
     }
 
     public BacktraceData toBacktraceData(Context context, Map<String, Object> clientAttributes) {
```

Comment:
there is one more thing which should be done, this method is used in the database (BacktraceDatabase.java) file, probably you will need to add `isProguardEnabled` parameter also there